app/main.py

```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    for person in people:
+        Person(person["name"], person["age"])
+
+    for person in people:
+        if "wife" in person and person["wife"]:
+            Person.people[person["name"]].wife = \
+                Person.people[person["wife"]]
+        elif "husband" in person and person["husband"]:
+            Person.people[person["name"]].husband = \
+                Person.people[person["husband"]]
```

Comment:
1. When you need to check this both conditions, you can use .get() method of dict that does the same thing
2. We don't use \ as a line breaker. It's better to use parenthesis
3. If you don't want to use \ and () you can, for example, move fetching person by name in a separate variable and reuse it because it's used in both if statements